singlestoredb/mysql/connection.py

```diff
@@ -490,13 +491,19 @@ def _config(key, arg):
         from .. import __version__ as VERSION_STRING
 
         self._connect_attrs = {
-            '_client_name': 'singlestoredb.mysql',
+            '_os': str(sys.platform),
             '_pid': str(os.getpid()),
+            '_client_name': 'singlestoredb Python',
```

Comment:
Is this a particular naming convention? I find the mixed casing a bit odd. I would think it should either be 'SingleStoreDB Python' or 'singlestoredb python'.

Reply:
It's odd indeed, will use the official name SingleStoreDB Python Client.